src/provisioner/provisioner.py

```diff
@@ -102,8 +102,8 @@ def config_apply(solution_config_url: str, cortx_conf_url: str = None,
             Log.error(f'Unable to load {solution_config_url} url, Error:{e}')
 
         # Secrets path from config file
-        if cortx_conf.get('cortx>common>storage>local'):
-            CortxProvisioner._secrets_path = cortx_conf.get('cortx>common>storage>local')+CortxProvisioner._rel_secret_path
+        if cortx_conf.get(f'{const.ROOT}>common>storage>local'):
```

Comment:
This will not work for both "conf" and "cortx". root should be a variable. 

```
root = None
map(lambda k: if Conf.key(k) != None: root=k, ['cortx', 'conf'])
```

Reply:
done